ant_world_test/ant_world_test.cc

```diff
@@ -136,25 +146,25 @@ int main()
             ant = !ant;
         }
 
-
         // Use right trigger to control forward movement speed
-        const float forwardMove = moveSpeed * deltaTime * joystick.getState(HID::JAxis::RightTrigger);
+        const meter_t forwardMove = moveSpeed * deltaTime * joystick.getState(HID::JAxis::RightTrigger);
 
         // Calculate movement delta in 3D space
-        x += forwardMove * sin(yaw * AntWorld::degreesToRadians) * cos(pitch * AntWorld::degreesToRadians);
-        y += forwardMove * cos(yaw * AntWorld::degreesToRadians) * cos(pitch * AntWorld::degreesToRadians);
-        z -= forwardMove * sin(pitch * AntWorld::degreesToRadians);
+        auto cosPitch = cos(pitch);
```

Comment:
How does this know that cos expects radians?

Reply:
The units lib has wrapper functions for (I think) all of the cmath functions, so you get a version of ``cos`` which takes a ``radian_t`` parameter and if you pass it a ``degree_t`` object it'll be implicitly cast and converted to radians. It's a pretty cool library!

Reply:
Damn that's cool!